packages/experiments/src/components/Card/Layout/Layout.tsx

```diff
@@ -0,0 +1,131 @@
+    return (
+      <div className={classNames.root}>
+        {headerElement}
+        <div className={classNames.contentAreaLayout}>{content}</div>
+        {footerElement}
+      </div>
+    );
+  }
+
+  private _generateContentElement(cardContentList: ICardContentDetails[]): JSX.Element[] {
+    const contentArea: JSX.Element[] = [];
+    // This works because we have priority is defined in enum as numbers if it is string this will not work
+    for (const priority in Priority) {
+      if (!isNaN(Number(priority))) {
+        cardContentList.map((cardContent: ICardContentDetails, i: number) => {
+          if (cardContent.priority.toString() === priority) {
+            switch (cardContent.cardContentType) {
+              case CardContentType.BodyText: {
+                const { subHeaderText, bodyText }: IBodyTextProps = cardContent.content as IBodyTextProps;
+                contentArea.push(<BodyText key={i} subHeaderText={subHeaderText} bodyText={bodyText} />);
+                break;
+              }
+              case CardContentType.ThumbnailList: {
+                const { thumbnailItems }: IThumbnailListProps = cardContent.content as IThumbnailListProps;
+                contentArea.push(<ThumbnailList key={i} thumbnailItems={thumbnailItems} />);
+                break;
+              }
+              case CardContentType.CompoundButtonStack: {
+                const { actions } = cardContent.content as ICompoundButtonStackProps;
+                contentArea.push(<CompoundButtonStack actions={actions} />);
+                break;
+              }
+            }
+          }
+        });
+      }
+    }
+
+    return contentArea;
+  }
+
+  private _generateHeader(header: ICardHeaderProps): JSX.Element | null {
+    if (header === null || header === undefined) {
+      return null;
+    }
+    return (
+      <CardHeader headerText={header.headerText} annotationText={header.annotationText} fontSize={header.fontSize} />
+    );
+  }
+
+  private _generateFooter(actions: IAction[], className: string): JSX.Element | null {
+    if (actions === null || actions === undefined) {
+      return null;
+    }
+    return (
+      <div id="actionBar" className={className}>
+        <ActionBar actions={actions} />
+      </div>
+    );
+  }
+
+  private _generateContentArea(
+    cardContentList: ICardContentDetails[],
+    contentLayoutClassName: string,
+    contentArea1ClassName: string,
+    contentArea2ClassName: string,
+    cardSize: CardSize
+  ): JSX.Element | null {
+    if (cardContentList === null || cardContentList === undefined) {
+      return null;
+    }
+
+    const contentAreaContents = this._generateContentElement(cardContentList);
+    if (contentAreaContents.length === 0) {
+      return null;
+    }
+
+    if (contentAreaContents.length > 1 && cardSize !== CardSize.small) {
+      return (
+        <div className={contentLayoutClassName}>
+          <div id="c1" className={contentArea1ClassName}>
```

Comment:
TSLint approves it.

Reply:
We probably still shouldn't do it. It can possibly cause conflicts with other components on the page.

---
In reply to: [195603810](https://github.com/OfficeDev/office-ui-fabric-react/pull/5196#discussion_r195603810) [](ancestors = 195603810)

Reply:
removed it

---
In reply to: [195605171](https://github.com/OfficeDev/office-ui-fabric-react/pull/5196#discussion_r195605171) [](ancestors = 195605171,195603810)